crates/starknet-devnet-server/src/builder.rs

```diff
@@ -95,11 +96,13 @@ impl<TJsonRpcHandler: RpcHandler, THttpApiHandler: Clone + Send + Sync + 'static
     pub fn build(self, starknet_config: &StarknetConfig) -> ServerResult<StarknetDevnetServer> {
         let mut svc = self.routes;
 
+        println!("DEBUG setting request body size: {}", starknet_config.request_body_size_limit);
```

Comment:
This print statement will be shown on each startup. Why not log It with `debug!` macro

Reply:
lol, I accidentally left it and didn't see it in the self-review, thanks for noticing

Reply:
Removed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/410/commits/993e799a5f087eb72f3a589c3a2fee2a606deeda